.changeset/tall-jars-carry.md

```diff
@@ -0,0 +1,5 @@
+---
+"sit-onyx": minor
+---
+
+feat(OnyxTable): initialize table component + add example
```

Comment:
I am not sure if we should release this since its nothing we want users to actually use, its just preparational work :)

Reply:
ok :)